File: napakalaki(ruby)/lib/prize.rb
#encoding: utf-8
# To change this license header, choose License Headers in Project Properties.
# To change this template file, choose Tools | Templates
# and open the template in the editor.
module NapakalakiGame

class Prize
  attr_reader :treasures, :levels
  def initialize(t, l)
    @treasures=t
    @levels=l
  end
  
  
  def to_s
   "\n Tesoros ganados : #{@treasures} \n Niveles ganados: #{@levels} \n"
  end
  
end
end

File: napakalaki/src/Napakalaki/Prize.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package Napakalaki;

/**
 *
 * @author marcos
 */
public class Prize {
    private int treasure;
    private int level;
       
      
    public Prize(int _treasure, int _level){
            treasure=_treasure;
            level=_level;
        }
        
    public int getTreasure(){
            return treasure;
        }
        
    public int getLevel(){
            return level;
        }

    public String toString(){
        return "Numero de tesoros = " + Integer.toString(treasure) + " "
                + " Niveles obtenidos = " + Integer.toString(level)+" "
                + "\n";

    }
        
    
}
